README.md

```diff
@@ -1,14 +1,71 @@
-SQL Server Extension for Azure Functions
-===
+# Azure SQL Extension for Azure Functions #
 
-This repository contains extension code for SQL Server input and output bindings. Trigger bindings are coming soon.
+## Introduction ##
 
-# Input and Output Binding
-## Quick Start
+This repository contains extension code for the Azure SQL trigger and bindings as well as a quick start, tutorial, and samples of how to use them. A high level explanation of the trigger and bindings is provided below. Additional information for each is in their respective sample sections.
 
-### .NET Function App
+- **input binding**: takes a SQL query to run on a provided table and returns the output of the query.
+- **output binding**: takes a list of rows and upserts them into the user table (i.e. If a row doesn't already exist, it is added. If it does, it is updated).
+- **trigger**: requires the user to specify the name of a table, and in the event a change occurs (i.e. the row is updated, deleted, or inserted), the trigger will return the updated rows and values along with any associated metadata.
 
-1. Add MyGet package feed.
+## Table of Contents ##
+
+- [Quick Start](#Quick-Start)
+  - [SQL Setup](#SQL-Setup)
+  - [Set Up Local .NET Function App](#Set-Up-Local-.NET-Function-App)
+- [Tutorials](#Tutorials)
+  - [Create Azure SQL Database](#Create-Azure-SQL-Database)
+  - [Input Binding Tutorial](#Input-Binding-Tutorial)
+  - [Output Binding Tutorial](#Output-Binding-Tutorial)
+  - [Trigger Tutorial](#Trigger-Tutorial)
+- [Samples](#Samples)
+  - [Input Binding](#Input-Binding)
+  - [Output Binding](#Output-Binding)
+  - [Trigger](#Trigger)
+
+## Quick Start ##
+
+### SQL Setup ###
+
+This requires already having a SQL database. If you need to create a SQL database, please refer to [Create Azure SQL Database](#Create-Azure-SQL-Database) in the tutorials section.
+
+A primary key must be set in your SQL table before using the bindings. To do this, run the below SQL commands in the SQL query editor.
+
+1. Ensure there are no NULL values in the primary key column. The primary key will usually be an ID column.
+
+    ```sql
+    ALTER TABLE ['your table name'] alter column ['column to be primary key'] int NOT NULL
```

Comment:
I don't know that Azure SQL (or whatever) supports tables without Primary Keys and/or null Primary Keys. IMO, it's unnecessary to include these instructions but I don't really have a strong opinion on it

Reply:
(referring to both this instruction and the `Set primary key column` instruction)

Reply:
I believe when I was initially testing out the SQL bindings the input binding didn't require me to set the primary keys/null keys for the table. However, enabling change tracking required it. I could be misremembering though. I think we can leave it in at least for the Bug Bash tomorrow and get feedback. I can test to double check this next week.